lib/hanami/cli/generators/app/migration.rb

```diff
@@ -17,17 +17,13 @@ def initialize(fs:, inflector:, out: $stdout)
 
           # @since 2.2.0
           # @api private
-          def call(_app_namespace, name, slice, **_opts)
-            normalized_name = inflector.underscore(name)
-            ensure_valid_name(normalized_name)
+          def call(namespace:, key:, base_path:, **_opts)
+            _namespace = namespace
```

Comment:
What's the reason for this extra assignment? It doesn't look like we use `_namespace` anywhere?

Reply:
Ah, it was to appease Rubocop, but clearly the better move is to just remove the `namespace:` arg altogether, which is what I just committed.